### PR TITLE
Lokalize: Init at 19.12.3

### PR DESCRIPTION
**Pull request by SCOTT-HAMILTON**
###### Motivation for this change
Needed an alternative to QtLinguist.

###### Things done
Packaged KDE Lokalize 19.12.3 (works also on 20.04.1).

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -20397,7 +20397,7 @@ in
     akonadi akregator ark dolphin dragon elisa ffmpegthumbs filelight gwenview k3b
     kaddressbook kate kcachegrind kcalc kcharselect kcolorchooser kdenlive kdf kdialog
     keditbookmarks kfind kget kgpg khelpcenter kig kleopatra kmail kmix kmplot kolourpaint kompare konsole yakuake
-    kpkpass kitinerary kontact korganizer krdc krfb ksystemlog ktouch kwalletmanager marble minuet okular spectacle;
+    kpkpass kitinerary kontact korganizer krdc krfb ksystemlog ktouch kwalletmanager lokalize marble minuet okular spectacle;
```
Comment (ttuegel) [inline pkgs/top-level/all-packages.nix:20400]: Please break long lines at 80 columns.

File: pkgs/applications/kde/lokalize.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  name = "lokalize";
+
+  nativeBuildInputs = [ pkg-config extra-cmake-modules cmake  ];
+
+  buildInputs = [ kross qtscript hunspell kactivities qtbase ];
+
+  meta = with lib; {
+    description = "Computer-aided translation system that focuses on productivity and quality assurance";
+    license = licenses.mit;
+    homepage = "https://github.com/KDE/lokalize";
```
Comment (ttuegel) [inline pkgs/applications/kde/lokalize.nix:24]:
```suggestion
    homepage = "https://kde.org/applications/en/development/org.kde.lokalize";
```

File: pkgs/applications/kde/lokalize.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  name = "lokalize";
+
+  nativeBuildInputs = [ pkg-config extra-cmake-modules cmake  ];
+
+  buildInputs = [ kross qtscript hunspell kactivities qtbase ];
+
+  meta = with lib; {
+    description = "Computer-aided translation system that focuses on productivity and quality assurance";
+    license = licenses.mit;
```
Comment (ttuegel) [inline pkgs/applications/kde/lokalize.nix:23]:
```suggestion
    license = licenses.gpl2Plus;
```

**Review by ttuegel — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (SCOTT-HAMILTON) [reply pkgs/top-level/all-packages.nix]: Done

**Comment by Mindavi** [comment]
It appears to me that lokalize 21.08.3 is now available in nixpkgs. Please reopen or resubmit if that's now what you're looking for.

**Pull request closed without merging**